README.md

```diff
@@ -15,14 +15,20 @@ Accept bitcoins on your WHMCS, payments go directly into your wallet
 - After setting API Key refresh page
 - Copy your Callback to Blockonomics Merchants > Settings
 
-## Screenshots ## 
+## Upgrade from 1.8.X to 1.9 ##
```

Comment:
Instructions are clear. These may be sent to upgrading users directly, and may not be required to be so prevalent on the readme. @shivaenigma can you advise on this.

Reply:
>Delete the folders css/ img/ js/ located in your WHMCS root installation. They are now located inside the module

I am concerned this may delete some of the other files that are there in WHMCS.

Reply:
Not from WHCMS, those folders are not part of WHMCS and should never existed there, that's was one of the main reason I propose to refactor the code. 

Impossible to deterine if any other plugin made the same mistake, but that is considered out of scope. The most you can do is extend the upgrade the instructions to warn users about that posibility.